.travis.yml

```diff
@@ -48,17 +49,18 @@ before_script:
     - ccache -s || true
 
 script:
-    # Run pycodestyle on all .py files in all subfolders
+    # Run black and pydocstyle on all .py files in all subfolders
     - black --check .
     - find . -type d -o ! -name versioneer.py ! -name test_*.py ! -name _version.py -name \*.py -exec pydocstyle --convention=numpy --add-ignore=D105,D202 {} +
 
     # Run tests
     - export PYTHONPATH=.:$PYTHONPATH
     - cd tests
-    - pytest -s
+    - pytest -x
```

Comment:
Do you still want to keep this change to pytest?

Reply:
Yes, why not?